src/plugins/options/option/option.less

```diff
@@ -1,47 +1,37 @@
 uip-option {
     display: block;
+    width: 1.5rem;
+    height: 1.5rem;
     margin: 0 0.5rem;
-    color: @option-color;
+    background-position: center;
+    background-repeat: no-repeat;
     cursor: pointer;
 
-    &.rtl-direction-option,
-    &.editor-collapsed-option.active,
-    &.settings-collapsed-option.active {
-        stroke-width: 0.1;
-        fill-opacity: 0.5;
-    }
-
-    &.rtl-direction-option.active,
-    &.editor-collapsed-option,
-    &.settings-collapsed-option {
-        stroke-width: 0.7;
-        fill-opacity: 1;
-    }
-
-    &.dark-theme-option {
-        svg {
-            stroke: none;
-        }
-
-        &:not(.active) .sun-icon {
-            display: none;
+    each(@option-icons, {
+      &.@{key}-option {
+        background-image: extract(@value, 1);
+        &.active {
+          background-image: extract(@value, 2);
         }
+      }
+    });
 
-        &.active .moon-icon {
-            display: none;
-        }
+    &.rtl-direction-option {
+      width: 2.5rem;
     }
 
-    esl-image {
-        display: block;
-        height: 100%;
-        width: 100%;
-
-        svg {
-            width: 100%;
-            vertical-align: middle;
-            fill: currentColor;
-            stroke: currentColor;
-        }
+    &.editor-collapsed-option {
+      width: 2rem;
     }
 }
+
+@option-icons: {
```

Comment:
Basically almost all of the icons are not optimized, you can save a lot of KB to users if you optimes the assets.